Section3/Exercise7/Main7.cpp

```diff
@@ -0,0 +1,92 @@
+/*Exercise 7 - Lottery
+Write a program that will simulate the lottery.Create two arrays of 5 integers.The first array should be populated with 5 random numbers from 1 - 20. For the second array, ask the user to input 5 numbers from 1 - 20. Make sure to handle invalid input and make sure numbers are not repeated.
+For each number that is matched in the same index, give the user $10.*/
+#include<iostream>
+#include <cstdlib>    
+#include <ctime>  
+using namespace std;
+bool repeatValues(int arrRepeat[], int size) {
+	for (int pos = 1; pos < size; pos++)
+	{
+		for (int posL = 0; posL < pos; posL++)
+		{
+			if (arrRepeat[pos] == arrRepeat[posL])
+			{
+				return true;
+			}
+		}
+	}
+	return false;
+}
+int main()
+{
+	int ArrayLotery[5];
+	int ArrayUser[5];
+	int ArrLoterySize = sizeof(ArrayLotery) / sizeof(ArrayLotery[0]);
+	int ArrUserSize = sizeof(ArrayUser) / sizeof(ArrayUser[0]);
+
+	cout << "Welcome to the lottery!" << endl;
+
+
+	int UserValues;
+
+	for (int UserSpace = 0; UserSpace < ArrUserSize; UserSpace++)
+	{
+		int entry = UserSpace + 1;
+		cout << "Please enter your entry number #" << entry << "(1-20)" << endl;
+		cin >> UserValues;
+		for (int pruebaU = 0; pruebaU < ArrUserSize; pruebaU++)
```

Comment:
Careful with the names, seems like you forgot to change some test

Reply:
the user only wins if he has the same numbers in the same boxes of the array, which is why I did it with several if's and 
 so I control that I can win more than 10
![Ifs](https://user-images.githubusercontent.com/36427649/179554431-fb9982b7-9126-48ea-afa9-ae91b66d9900.PNG)
f